### PR TITLE
Add multivisor recipe

### PR DESCRIPTION
**Pull request by beenje**
Checklist
- [X] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [X] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [X] Source is from official source.
- [X] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [ ] If static libraries are linked in, the license of the static library is packaged.
- [X] Build number is 0.
- [X] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [X] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [ ] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by beenje** [comment]
Build is failing on windows because `supervisor` isn't available, but package is noarch.

@conda-forge/help-python, ready for review!

File: recipes/multivisor/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+  - name: multivisor-web
+    build:
+      noarch: python
+    requirements:
+      host:
+        - pip
+        - python >=3.6
+      run:
+        - python >=3.6
+        - {{ pin_subpackage('multivisor', exact=True) }}
```
Comment (jaimergp) [inline recipes/multivisor/meta.yaml:49]: I think there were some issues with `exact=True`, multi output recipes and noarch, which might require you to use `max_pin='x.x.x.x'`, instead, but I can't recall what was the problem :/

File: recipes/multivisor/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+      commands:
+        - multivisor-cli --help
+
+about:
+  home: https://github.com/tiagocoutinho/multivisor
+  summary: A centralized supervisor UI (web & CLI)
+  license: GPL-3.0-or-later
+  license_file: LICENSE
+
+extra:
```
Comment (jaimergp) [inline recipes/multivisor/meta.yaml:105]: You might want to add `feedstock-name` here to make sure you get a `multivisor-feedstock` repo and not a `multivisor-suite-feedstock` repo, if that's preferred.

File: recipes/multivisor/meta.yaml (hunk 1 above)
Comment (beenje) [reply recipes/multivisor/meta.yaml]: I think I had to put the same host requirement everywhere to get the same build string.
I don't see any issue using max_pin. I changed to that.

**Comment by beenje** [comment]
Nice catch about the `tests`! I added a patch and created a PR in the upstream repo: https://github.com/tiagocoutinho/multivisor/pull/90

Thanks!

File: recipes/multivisor/meta.yaml (hunk 2 above)
Comment (jaimergp) [reply recipes/multivisor/meta.yaml:107]:
```suggestion
extra:
  feedstock-name: multivisor
```

Then I think we need this ^

**Pull request merged**